scripts/ci-check-to-ensure-sql-is-in-lowercase.mjs

```diff
@@ -0,0 +1,98 @@
+import fs from "fs";
+import path from "path";
+
+const regex = /```sql\n([\s\S]*?)\n```/g;
+const keywords = [
+  "SELECT",
+  "UPDATE",
+  "INSERT",
+  "DELETE",
+  "WITH",
+  "FROM",
+  "WHERE",
+  "ORDER BY",
+  "GROUP BY",
+  "HAVING",
+  "AS",
+  "VALUES",
+];
+
+const WEBSITE_FOLDERS = [
+  "../postgraphile/website",
+  "../grafast/website",
+  "../graphile-build/website",
+  "../utils/website",
+];
+
+async function walkDir(dir) {
+  const files = await fs.promises.readdir(dir, { withFileTypes: true });
+  let result = [];
+
+  for (let file of files) {
+    const fullPath = path.join(dir, file.name);
+    if (file.isDirectory()) {
+      result = result.concat(await walkDir(fullPath));
+    } else {
+      result.push(fullPath);
+    }
+  }
+  return result;
+}
+
+async function readFileAsync(filePath) {
+  try {
+    const data = await fs.promises.readFile(filePath, "utf-8");
+    return data;
+  } catch (error) {
+    console.error("Error reading file:", error);
```

Comment:
This logs the error but continues anyway; would it be better to throw the error to ensure that any such issue is fixed?
```suggestion
    console.error(`Error reading file ${filePath}:`, error);
    throw error;
```